docs/digger/installation/run-digger-installer.asciidoc

```diff
@@ -1,22 +1,22 @@
-== Run the Digger Installer
+== Running the Digger Installer
 
 .To run the Digger installer, follow these steps:
 
 . Clone the https://github.com/aerogear/digger-installer[Digger Installer GitHub repo^]
 
 . Make a copy of the Ansible `inventory-sample` file from the repo, and update it with the information about your infrastructure
 +
-NOTE: For a full list of variables that can be configured via the inventory file see the link:#installer-variables[Installer Variables] section.
+NOTE: For a full list of variables that can be configured via the inventory file refer to link:#installer-variables[Installer Variables].
 
-. Run the playbook using the updated `inventory-sample` and follow the instructions printed in the terminal.:
+. Run the playbook using the updated `inventory-sample` and follow the instructions printed in the terminal:
 
-** Command to install digger without a macOS server
+** Command to install Digger without a macOS server
 +
 ----
 ansible-playbook -i <your-inventory-sample> sample-build-playbook.yml --skip-tags=osx-provision
```

Comment:
@laurafitzgerald @matskiv i believe osx-provision does not exist in the code.
provision-osx should be ok. wdyt

Reply:
resolved